packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -312,23 +235,58 @@ function NetworkReady({ children }: { children: React.ReactNode }) {
     )
   }, [childChain.id, parentChain.id])
 
+  if (!tosAccepted) {
+    return (
+      <>
+        <Header />
+        <WelcomeDialog />
```

Comment:
welcome dialog is not a `<Dialog/>` now

<img width="1222" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/38a21b46-c5f7-4a9b-8d2f-1f765ff8073b">

`<Header/>` is still shown for mobile header